src/slice_container.rs

```diff
@@ -17,7 +17,7 @@ unsafe impl Send for PySliceContainer {}
 impl<T: Send> From<Box<[T]>> for PySliceContainer {
     fn from(data: Box<[T]>) -> Self {
         unsafe fn drop_boxed_slice<T>(ptr: *mut u8, len: usize, _cap: usize) {
-            let _ = Box::from_raw(slice::from_raw_parts_mut(ptr as *mut T, len) as *mut [T]);
+            let _ = Box::from_raw(ptr::slice_from_raw_parts_mut(ptr as *mut T, len));
```

Comment:
I just noticed this function is using mem::forget. It should be using ManuallyDrop, see https://doc.rust-lang.org/std/mem/fn.forget.html#relationship-with-manuallydrop . I'll try to fix these later.

Reply:
There is nothing in there that could panic and lead to double free. We discussed both options when those functions were written and settled on `mem::forget` for readability (`ManuallyDrop` was on table already then).